src/System.Collections.Concurrent/ref/System.Collections.Concurrent.cs

```diff
@@ -64,6 +64,9 @@ public partial class ConcurrentBag<T> : System.Collections.Concurrent.IProducerC
         bool System.Collections.ICollection.IsSynchronized { get { throw null; } }
         object System.Collections.ICollection.SyncRoot { get { throw null; } }
         public void Add(T item) { }
+#if netcoreapp11
```

Comment:
I've not updated the guide yet. I just filed https://github.com/dotnet/corefx/issues/15323 to track that. However as a quick summary now we will not be building netstandard refs in corefx any more for the contracts that are part of netstandard, those will be built in the dotnet/standard repo. Which means when editing those here the default will be netcoreapp and so you won't need these #ifdef's any longer and you will just naturally add them.